examples/messaging/messaging.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Can you add comments to this?
I have no idea what it is an example of

Reply:
Correct me if I am wrong but I think that comments are not available in JSON format.
This file is an example of l1 messaging worker configuration (renamed file).